src/Graph/SubGraph.ts

```diff
@@ -193,8 +193,9 @@ export default class SubGraph {
     const mosaicConfig: MosaicConfig = MosaicConfig.fromChain(this.originChain);
     return {
       projectRoot: Directory.projectRoot,
-      ostComposerAddress: mosaicConfig.originChain.contractAddresses.ostComposerAddress,
-      eip20GatewayAddress: mosaicConfig.auxiliaryChains[this.auxiliaryChain].contractAddresses.origin.ostEIP20GatewayAddress,
+      stakePoolAddress: mosaicConfig.originChain.contractAddresses.stakePoolAddress,
+      baseTokenAddress: mosaicConfig.originChain.contractAddresses.valueTokenAddress,
```

Comment:
why do we need `baseTokenAddress ` here ? I guess it is not being used in sub graph deployment

Reply:
Agree! reverted the change.